css/style.css

```diff
@@ -0,0 +1,369 @@
+  margin-bottom: 60px;
+  font-size: 18px;
+  color: #FFFFFF;
+}
+
+.level-choice__wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-content: center;
+  align-items: start;
+  max-height: 300px;
+}
+
+.level {
+  position: relative;
+  display: block;
+  margin-bottom: 24px;
+  font-size: 48px;
+  line-height: 59px;
+  color: #FFFFFF;
+  margin-left: 46px;
+}
+
+.selected-level {
+  color: #D62B2E;
+}
+
+.level::before {
+  display: none;
+  content: '';
+  position: absolute;
+  top: 20px;
+  right: 102%;
+  width: 18px;
+  height: 18px;
+  background: #D62B2E;
+  transform: rotate(45deg);
+}
+
+.level:hover::before {
+  display: block;
+}
+
+button {
+  margin-top: 88px;
+  min-width: 352px;
+  min-height: 95px;
+  background: #D62B2E;
+  border-radius: 7px;
+  font-size: 32px;
+  color: #FFFFFF;
+}
+
+.first-card {
+  position: absolute;
+  left: 60%;
+  top: 35%;
+  transform: rotate(11.9deg);
+  z-index: -2;
+}
+
+.second-card {
+  position: absolute;
+  left: 69%;
+  top: 45%;
+  transform: rotate(39.25deg);
+  z-index: -1;
+}
+
+.hide {
+  display: none;
+}
+
+.table {
+  display: none;
+  width: 700px;
+  align-items: start;
+}
+
+.table-0 {
+  display: flex;
+  height: 35%;
+  justify-content: space-between;
+}
+
+.table-1 {
+  display: flex;
+  height: 94%;
+  max-height: 720px;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.table-2 {
+  display: flex;
+  justify-content: center;
+  width: 600px;
+  height: 95%;
+  flex-wrap: wrap;
+}
+
+.flip-card {
+  width: 31%;
+  text-align: center;
+  perspective: 1200;
+  -webkit-perspective: 1200;
+  -moz-transform: perspective(1200px);
+  transform-style: preserve-3d;
+  -webkit-transform-style: preserve-3d;
+  -moz-transform-style: preserve-3d; 
+}
+
+.table-2 .flip-card {
+  width: 26%;
+}
+
+.table-2 .card-inner {
+  width: 99%;
+}
+
+.table-2 .card, .table-2 .result {
+  width: 93%;
+}
+
+.card-inner {
+  position: relative;
+  width: 100%;
+  transform-style: preserve-3d;
+  -webkit-transform-style: preserve-3d;
+  -moz-transform-style: preserve-3d;
+  transition: .7s linear;  
+  -webkit-transition: .7s linear;
+  -moz-transition: .7s linear;
+  -o-transition: .7s linear;
+}
+
+.front-side, .back-side {
+  position: absolute;
+  top: 0;
+  left: 0;
+  -webkit-backface-visibility: hidden;
+  -moz-backface-visibility: hidden;
+  backface-visibility: hidden;
+}
+
+.back-side {
+  -webkit-transform: rotateY(0deg);
+  -moz-transform: rotateY(0deg);
+  -ms-transform: rotateY(0deg);
+  transform: rotateY(0deg);
+}
+
+.front-side {
+  -webkit-transform: rotateY(-180deg);
+  -moz-transform: rotateY(-180deg);
+  -ms-transform: rotateY(-180deg);
+  transform: rotateY(-180deg);
+}
+
+.card, .result {
+  width: 100%;
+}
+
+.flip {
+  -webkit-transform: rotateY(180deg);
+  -moz-transform: rotateY(180deg);
+  -ms-transform: rotateY(180deg);
+  transform: rotateY(180deg);
+}
+
+@keyframes rotation {
+  from {transform: scale(1);}
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/bugs-finde/pullRequest?prid=5808908)